src/shared/lib/oql/oql_basic.pegjs

```diff
@@ -1,23 +1,25 @@
 start
-	= Query
+	= SuperQuery
 	/ sp { return false; }
 
-br	= [\n] b:br
-        / ";" b:br
-        / [\n]
-	/ ";"
-
 NaturalNumber = number:[0-9]+ { return number.join("");}
 Number = "-" number: Number { return "-"+number;}
         / whole_part:NaturalNumber "." decimal_part:NaturalNumber { return whole_part + "." + decimal_part;}
         / "." decimal_part:NaturalNumber { return "."+decimal_part;}
         / whole_part:NaturalNumber {return whole_part;}
 String = word:[-_.@/a-zA-Z0-9*]+ { return word.join("") }
 AminoAcid = letter:[GPAVLIMCFYWHKRQNEDST] { return letter; }
+AnyString = anyString:[^"]+ { return anyString.join("") }
```

Comment:
bad name, AnyString doesnt really explain difference with String on line 10. Maybe StringWithinQuotes or something?

Reply:
Good point, I'll change it.

Reply:
Changed to `StringExceptQuotes`. See last commit.